### PR TITLE
Fix: broken RTL code blocks

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

related: #198 

Basted on my testing this solutions should resolve the code breaks in the RTL layout; however, further testing is encouraged.

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
Tested with a local Arabic build and this appears to solve the issue of inline code breaking to new lines unexpectedly.

Here's the way inline code breaks currently:

image

And here it is after the fix:

image

One thing I noticed is that `word-break: break-word` seems to be deprecated. I'll leave a review comment with a possible alternative.

File: src/_includes/assets/css/screen.css [hunk 1]
```diff
@@ -2470,3 +2470,7 @@ a.banner:hover span {
   margin-right: 0;
   margin-left: 19px;
 }
+
+.rtl-layout .post-full-content p code {
+    word-break: break-word;
```
Comment (scissorsneedfoodtoo) [inline src/_includes/assets/css/screen.css]: This seems to be the fallback for `word-break: break-word` now that `break-word` is a deprecated value:

```suggestion
  word-break: normal;
```

Comment (ahmaxed) [inline src/_includes/assets/css/screen.css]:
```suggestion
  word-break: break-word; 
```

File: src/_includes/assets/css/screen.css [hunk 2]
```diff
@@ -2470,3 +2470,7 @@ a.banner:hover span {
   margin-right: 0;
   margin-left: 19px;
 }
+
+.rtl-layout .post-full-content p code {
+  word-break: break-word;
```
Comment (ahmaxed) [inline src/_includes/assets/css/screen.css]:
```suggestion
.rtl-layout .post-full-content p code {
  word-break: normal; 
```

**Review by scissorsneedfoodtoo — APPROVED** [review]
Sorry, I may have messed up the earlier review and suggestion.

Everything here LGTM :+1: :+1:

**Pull request merged**